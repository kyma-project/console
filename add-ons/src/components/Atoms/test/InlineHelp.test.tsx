import React from 'react';
import InlineHelp from './../InlineHelp';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('InlineHelp', () => {
  it('Renders with minimal props', () => {
    const component = shallow(<InlineHelp text="sample text" />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
