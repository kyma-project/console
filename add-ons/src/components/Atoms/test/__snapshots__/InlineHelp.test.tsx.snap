// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InlineHelp Renders with minimal props 1`] = `
<span
  className="fd-inline-help fd-has-float-right"
>
  <span
    className="fd-inline-help__content fd-inline-help__content--bottom-left"
  >
    sample text
  </span>
</span>
`;
